src/commands/npm/lerna/release.ts

```diff
@@ -152,8 +152,8 @@ export default class Release extends SfdxCommand {
   }
 
   protected async verifySign(pkgInfo: PackageInfo): Promise<void> {
-    const cmd = 'trust:plugins:verify';
-    const argv = `--npm ${pkgInfo.name}@${pkgInfo.nextVersion} ${pkgInfo.registryParam}`;
+    const cmd = 'plugins:trust:verify';
+    const argv = `--npm ${pkgInfo.name}@${pkgInfo.nextVersion} --registry ${pkgInfo.registryParam}`;
```

Comment:
`--registry` is already included in `pkgInfo.registryParam`
https://github.com/salesforcecli/plugin-release-management/blob/d35459ee5550007008f6967d0ab1eb5ef5a1df57/src/registry.ts#L28-L30

I mistyped the command when moving it from a dummy test command 🤦🏼‍♂️